.gitlab-ci.yml

```diff
@@ -355,14 +355,16 @@ gh-pages:
         -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DBUILD_SHARED_LIBS=ON
         ${EXTRA_CMAKE_FLAGS} -DGINKGO_DEVEL_TOOLS=OFF -DGINKGO_BUILD_REFERENCE=OFF
         -DGINKGO_BUILD_OMP=OFF -DGINKGO_BUILD_CUDA=OFF -DGINKGO_BUILD_TESTS=OFF
-        -DGINKGO_BUILD_EXAMPLES=OFF -DGINKGO_BUILD_DOC=ON
+        -DGINKGO_BUILD_EXAMPLES=OFF -DGINKGO_BUILD_DOC=ON -DGINKGO_DOC_GENERATE_PDF=ON
     - make usr
```

Comment:
I think you need to add a `make pdf ` step here to generate the pdf.

Reply:
Yes you are correct. This is what I was now doing, though there was also a problem with the containers (ghostscript is also required).